tests/scripts/pulp_ostree/test_modify_content.sh

```diff
@@ -16,35 +16,40 @@ expect_succ pulp ostree repository create --name "cli_test_ostree_repository1"
 expect_succ pulp ostree repository sync --name "cli_test_ostree_repository1" \
   --remote "cli_test_ostree_remote"
 
+expect_succ pulp ostree repository content list --repository cli_test_ostree_repository1 --type ref
+COMMIT_CHECKSUM1=$(echo "$OUTPUT" | jq -r ".[0].checksum")
+COMMIT_CHECKSUM2=$(echo "$OUTPUT" | jq -r ".[1].checksum")
+REF_NAME2=$(echo "OUTPUT" | jq -e ".[1]".checksum)
```

Comment:
Single quotes are usually the easier escaping for jq statements.
```suggestion
COMMIT_CHECKSUM1=$(echo "$OUTPUT" | jq -r '.[0].checksum')
COMMIT_CHECKSUM2=$(echo "$OUTPUT" | jq -r '.[1].checksum')
REF_NAME2=$(echo "$OUTPUT" | jq -r '.[1].name')
```